### DISCUSSION TITLE
Killed * DB saved on disk MaxListenersExceededWarning: (node) warning: possible EventEmitter memory leak detected. 11 listeners added. Use emitter.setMaxListeners() to increase limit. Event name: update.

### DISCUSSION THREAD

**Discussion by Jeyakumarrathnaselvi** [directus/directus#15100] [Archive]
### Describe the Bug

I am getting this error after 100+ PATCH request even with 1 second one item patched.

Aug 17 00:12:00 15:M 16 Aug 2022 18:41:57.409 * 10 changes in 300 seconds. Saving...
Aug 17 00:12:01 15:M 16 Aug 2022 18:41:59.466 * Background saving started by pid 525
Aug 17 00:12:03 Killed
Aug 17 00:12:03 525:C 16 Aug 2022 18:42:03.251 * DB saved on disk
Aug 17 00:12:03 525:C 16 Aug 2022 18:42:03.253 * RDB: 0 MB of memory used by copy-on-write
Aug 17 00:12:03 15:M 16 Aug 2022 18:42:03.266 * Background saving terminated with success
Aug 17 00:12:05 ==> Enabling postgis extension
Aug 17 00:12:05 NOTICE: extension "postgis" already exists, skipping
Aug 17 00:12:05 CREATE EXTENSION
Aug 17 00:12:05 ==> DB Migration
Aug 17 00:12:13 18:42:13 ✨ Running migrations...
Aug 17 00:12:13 18:42:13 ✨ Database up to date
Aug 17 00:12:13 ==> Starting Directus
Aug 17 00:12:17 (node:62) MaxListenersExceededWarning: (node) warning: possible EventEmitter memory leak detected. 11 listeners added. Use emitter.setMaxListeners() to increase limit. Event name: update.
Aug 17 00:12:17 (Use `node --trace-warnings ...` to show where the warning was created)
Aug 17 00:12:17 18:42:17 ✨ Server started at http://0.0.0.0:8055
Aug 17 00:12:18 18:42:18 ✨ request completed GET 304 /items/

![image](https://user-images.githubusercontent.com/36895131/184955771-c1bd4d9e-49c5-4ce5-86e3-0569467a2b7a.png)

### To Reproduce

PATCH runs continiously for 500 records with 1 second delay

### Errors Shown

Killed * DB saved on disk MaxListenersExceededWarning: 

### What version of Directus are you using?

v9.15.1

### What version of Node.js are you using?

16.13.1

### What database are you using?

Postgres

### What browser are you using?

Chrome

### How are you deploying Directus?

Cloudron

**Comment by rijkvanzanten** [comment]
I'm not sure how you're hosting this, but FWIW these:

```
Aug 17 00:12:00 15:M 16 Aug 2022 18:41:57.409 * 10 changes in 300 seconds. Saving...
Aug 17 00:12:01 15:M 16 Aug 2022 18:41:59.466 * Background saving started by pid 525
Aug 17 00:12:03 Killed
Aug 17 00:12:03 525:C 16 Aug 2022 18:42:03.251 * DB saved on disk
Aug 17 00:12:03 525:C 16 Aug 2022 18:42:03.253 * RDB: 0 MB of memory used by copy-on-write
Aug 17 00:12:03 15:M 16 Aug 2022 18:42:03.266 * Background saving terminated with success
```

are logs from the database apparently, where something got killed for some reason, which is different from:

```
Aug 17 00:12:17 (node:62) MaxListenersExceededWarning: (node) warning: possible EventEmitter memory leak detected. 11 listeners added. Use emitter.setMaxListeners() to increase limit. Event name: update.
```

which is just node warning for a lot of event listeners (which is not an issue here, as Directus relies on event listeners for hooks/flows/etc 🙂). 11 is not an unconventional amount 👍🏻 

It sounds to me like the large amount of requests is killing your database 🤔

**Reply by Jeyakumarrathnaselvi** [reply]
we are not doing any direct postgres activity. we use directus api to access postgres. 

we hit directus  api 1 second once only.

we deploy it through cloudron

**Reply by rijkvanzanten** [reply]
Don't know! That `Killed` could mean a range of things. Oftentimes it indicates that whatever server you're using force quit the process because it was using too much CPU or memory. That could either be the database or the directus process, hard to tell from the shared logs 🤷🏻
